.github/workflows/release-snapshot.yml

```diff
@@ -12,15 +12,23 @@ on:
 jobs:
   snapshot-snapshot:
     name: Publish snapshot release to npm
-    if: github.repository == 'ethereum-optimism/optimism'
```

Comment:
GH actions if statement cannot be parametrized hence the hack below that does the same :)